CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 All notable changes for the LINZ BDE schema are documented in this file.
 
 ## 1.8.0dev - 2019-MM-DD
-### Added
+### Enhanced
+- Schema loaders will now recover permissions (#165)
```

Comment:
* @strk , would you please change this changelog message to something more clear ? Thanks

Reply:
Little bit better.